recipe/run_test.py

```diff
@@ -86,17 +86,20 @@
     import syslog
     import termios
 
+    if os.getenv('PY_INTERP_DEBUG') == 'yes':
+        assert 'd' in sys.abiflags
+        assert 'gettotalrefcount' in dir(sys)
```

Comment:
Can you check this for windows too?

Reply:
Please see https://github.com/conda-forge/python-feedstock/pull/597#issuecomment-1314960208. `assert 'd' in sys.abiflags` only works for POSIX systems. This is also why debug builds are currently skipped for windows. Not sure if the other test (`'gettotalrefcount' in dir(sys)`) works on windows though.

Reply:
`sys.abiflags` does not change in windows because the headers are not configured unlike in unix. However `sys.gettotalrefcount` is a good test.